plugin.rb

```diff
@@ -575,4 +575,28 @@ def self.enqueue_assign_hooks(event, payload)
       end
     end
   end
+
+  if defined?(DiscourseAutomation)
+    add_automation_scriptable('random_assign') do
+      field :assignees_group, component: :group
+      field :assigned_topic, component: :text
+
+      version 1
+
+      triggerables %i[point_in_time recurring]
+
+      script do |context, fields|
+        next unless SiteSetting.assign_enabled?
+
+        next unless group_id = fields.dig('assignees_group', 'group_id')
+        next unless group = Group.find_by(id: group_id)
+        assign_to = group.group_users.order(Arel.sql('RANDOM()')).first.user
```

Comment:
This is already merged so it doesn't matter much, just curious why you used `Arel.sql('RANDOM()')` instead of just `'RANDOM()'` inside `order()` here @jjaffeux ?

Reply:
> This is already merged so it doesn't matter much, just curious why you used `Arel.sql('RANDOM()')` instead of just `'RANDOM()'` inside `order()` here @jjaffeux ?

https://stackoverflow.com/questions/48897070/deprecation-warning-dangerous-query-method-random-record-in-activerecord-5

Is it not the case anymore?

Reply:
I didn’t see any warnings when I tried it in rails console @jjaffeux .  Weird.  Anyway no need to change was just curious, thanks for the link!